src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -37,127 +38,104 @@
 			<RowDefinition Height="36" />
 			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
-		<Grid.ColumnDefinitions>
-			<ColumnDefinition Width="Auto" />
-			<ColumnDefinition Width="*" />
-		</Grid.ColumnDefinitions>
 
 		<!--  Titlebar  -->
-		<Border
-			x:Name="TitlebarArea"
-			Grid.ColumnSpan="2"
-			Height="36"
-			Background="Transparent">
-			<StackPanel
-				Padding="12,0"
-				Orientation="Horizontal"
-				Spacing="8">
-
-				<Image
-					Width="20"
-					Height="20"
-					Source="ms-appx:///Assets/AppTiles/Dev/Square44x44Logo.png" />
-
-				<TextBlock
-					VerticalAlignment="Center"
-					Style="{StaticResource CaptionTextBlockStyle}"
-					Text="{helpers:ResourceString Name=Properties}" />
-
-			</StackPanel>
+		<Border x:Name="TitlebarArea" Height="40">
+			<TextBlock
+				Padding="16,0"
+				VerticalAlignment="Center"
+				Style="{StaticResource BodyStrongTextBlockStyle}"
+				Text="{helpers:ResourceString Name=Properties}" />
 		</Border>
 
-		<!--  Vertical NavigationView whose each items are squared  -->
+		<!--  Main NavigationView  -->
 		<NavigationView
 			x:Name="MainPropertiesWindowNavigationView"
 			Grid.Row="1"
-			Width="80"
 			IsBackButtonVisible="Collapsed"
 			IsPaneToggleButtonVisible="False"
 			IsSettingsVisible="False"
 			IsTitleBarAutoPaddingEnabled="False"
 			MenuItemsSource="{x:Bind NavViewItems, Mode=TwoWay}"
-			OpenPaneLength="88"
+			OpenPaneLength="180"
 			PaneDisplayMode="Left"
 			SelectionChanged="MainPropertiesWindowNavigationView_SelectionChanged">
 
 			<NavigationView.MenuItemTemplate>
-				<DataTemplate x:DataType="local:SquareNavViewItem">
-					<NavigationViewItem
-						Height="64"
-						IsSelected="{x:Bind IsSelected, Mode=TwoWay}"
-						Tag="{x:Bind ItemType}">
+				<DataTemplate x:DataType="local:NavigationViewItemButtonStyleItem">
+					<NavigationViewItem IsSelected="{x:Bind IsSelected, Mode=TwoWay}" Tag="{x:Bind ItemType}">
 						<NavigationViewItem.Content>
-							<StackPanel Spacing="4">
+							<Grid>
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition Width="Auto" />
+									<ColumnDefinition Width="*" />
+								</Grid.ColumnDefinitions>
 
 								<!--  Icon  -->
 								<Viewbox
-									Width="24"
-									Height="24"
+									Width="16"
+									Height="16"
 									VerticalAlignment="Center">
-									<usercontrols:OpacityIcon IsSelected="{x:Bind IsSelected, Mode=OneWay}" Style="{x:Bind OpacityIconStyle, Mode=OneWay}" />
+									<uc:OpacityIcon IsSelected="{x:Bind IsSelected, Mode=OneWay}" Style="{x:Bind OpacityIconStyle}" />
```

Comment:
```suggestion
									<uc:OpacityIcon IsSelected="{x:Bind IsSelected, Mode=OneWay}" Style="{x:Bind OpacityIconStyle, Mode=OneWay}" />
```

Reply:
We always define mode

Reply:
Can you explain why for me? I'd like to understand.

Reply:
It makes the xaml more readable

Reply:
I definitely recommend we should not do that in DataTemplate. This cause a warning `WMC1506: OneWay bindings require at least one of their steps to support raising notifications when their value changes`.

Reply:
Template could be an exception for `OneWay` but I believe it's needed for `TwoWay`.

Reply:
Yes, if a data have to be needed two way bind, have to add `Mode=TwoWay`. But should not add `Mode=OneWay` in one way binding. (In DataTemplate)

Reply:
Default Mode is OneTime in x:Bind. OpacityIconStyle is being set in its class's constructor so I believe it's not needed. But your decision actually whether we should add OneWay for each binds in TwoWay template.

Reply:
I think it's better for readability and forces developers to double check that they set the correct mode.